### PR TITLE
Add error response example for quota usage

### PR DESCRIPTION
**Pull request by daallen303**

### CODE REVIEW

**Review by heymatthew — APPROVED** [review]
LGTM

File: src/fiat/funds-transfers.md [hunk 1]
```diff
@@ -62,6 +62,7 @@ curl -X POST "https://service.centrapay.com/api/topups" \
 | 403    | {% break _ BANK_AUTHORITY_WALLET_MISMATCH %} | The wallet and the bank authority for the top up request do not belong to the same account. |
 | 403    | {% break _ MAX_INFLIGHT_TOPUPS_EXCEEDED %}   | The bank authority already has ten pending top ups, which is the maximum a bank authority can have at any one time. |
 | 403    | {% break _ MAX_INFLIGHT_TOPUP_AMOUNT_EXCEEDED %} | The top up can not be created because it is above the max amount of funds a bank authority can have pending at any one time. The max amount is $1000.00 for verified bank authorities and $100.00 for non-verified bank authorities. |
+| 403    | {% break _ QUOTA_EXCEEDED %} | The topup exceeds one or more topup quota limits see [ Quota Error Response ](../quotas.html#quota-error-response)  |
```
Comment (heymatthew) [inline src/fiat/funds-transfers.md:65]: If you do this...

```{% link /quotas.md %}#quota-error-response```

Then it'll fail the build when the page moves.

File: src/quotas.md [hunk 2]
```diff
@@ -99,3 +99,22 @@ applicable to the quota.
 |:------|:--------------------------------------------------------------------|:------:|:-------------------|
 | spend | Value of wallet-based payments or asset transfers from the account. |   Yes  | `monthly` `yearly` |
 | topup | Value of topups from bank accounts linked to the account.           |   Yes  | `monthly` `yearly` |
+
+<span id="quota-error-response">
```
Comment (heymatthew) [inline src/quotas.md:103]: Could we close the span tag?

Could we use the word `-anchor` as a postfix to make it clear that this is for linking to?

**Review by heymatthew — COMMENTED** [review]
LGTM, just little things that'd make this better.

File: src/quotas.md (hunk 2 above)
Comment (daallen303) [reply src/quotas.md]: Done :)

File: src/fiat/funds-transfers.md (hunk 1 above)
Comment (daallen303) [reply src/fiat/funds-transfers.md]: 👍

Comment (daallen303) [reply src/fiat/funds-transfers.md]: Done

**Pull request merged**